pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/Worker.java

```diff
@@ -38,6 +50,13 @@
     private final WorkerService workerService;
     private WorkerServer server;
 
+    private ZooKeeperClientFactory zkClientFactory = null;
+    private final OrderedExecutor orderedExecutor = OrderedExecutor.newBuilder().numThreads(8).name("pulsar-ordered").build();
```

Comment:
why it is hardcoded to `8`? we should make it configurable.

`pulsar-ordered` isn't a good name. if this is only used for zk cache, why not just call it `zk-cache-ordered`? since the `cacheExecutor` is called `zk-cache-callback`.

Reply:
@sijie I just replicated what is already done in PulsarService.java

https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java#L143

Reply:
I will rename the thread pool